.circleci/config.yml

```diff
@@ -461,6 +450,12 @@ jobs:
           name: Comment on PR and upload visual test images if any produced
           command: echo ${CIRCLE_PULL_REQUEST##*/} | xargs -I{} npx gulp update-pr-testresults-cypress --fail-silently --user circleci-mu --pr {}
           when: always
+      - run:
```

Comment:
In this case I would suggest to move

```
- run:
          name: Write file sizes at master
          command: npx gulp write-file-sizes --filename master.json
```
above moved lines instead of leaving them on top.

@goransle what do you think?

Reply:
as in doing another round of checkouts at the end of the job?

Reply:
I mean, call "writes size to master.json" before the file comparison instead of running write first, then call some other things and then file comparison.